src/script.js

```diff
@@ -1,50 +1,33 @@
 import './style.css';
-
-const toDoList = [
-  {
-    index: '1',
-    description: 'Attend stand-up meeting by 5pm',
-    completed: 'false',
-  },
-
-  {
-    index: '1',
-    description: 'see car dealer',
-    completed: 'false',
-  },
-
-  {
-    index: '1',
-    description: 'Read Eloquent JavaScript',
-    completed: 'false',
-  },
-
-  {
-    index: '1',
-    description: 'Mark text script',
-    completed: 'false',
-  },
-
-];
+import TodoList from './addRemove.js';
 
 const taskList = document.querySelector('.task-container');
```

Comment:
- [ ] Kindly note that the edited text doesn't persist in the `localstorage` after refresh 🙃 as it is a requirement.